shared/src/commonMain/kotlin/org/hyperskill/app/main/presentation/AppReducer.kt

```diff
@@ -180,4 +173,25 @@ class AppReducer(
             }
         }.toSet()
     }
+
+    private fun getOnAuthActions(
+        profileId: Long
+    ): Set<Action> =
+        setOf(
+            Action.IdentifyUserInSentry(userId = profileId),
+            Action.UpdateDailyLearningNotificationTime,
+            Action.SendPushNotificationsToken
+        )
```

Comment:
@ivan-magda 
Is it possible to remove `notificationsRegistrationService.renewAPNsDeviceToken()` call from the application startup?
In the AppReducer the token is sent when the user is authorized

Reply:
We can't remove this call from the application startup, on iOS there are 2 tokens: APNs and FCM.

`notificationsRegistrationService.renewAPNsDeviceToken()` - initiates the registration process with APNs, on success another delegate method returns a device token, and then we pass this token along to the firebase cloud messaging, as a result fcm token could change.

Reply:
Then, we can't send `SendPushNotificationsToken` on application startup, because on the iOS side, the FCM token is sent twice.
Maybe it's better to call `notificationsRegistrationService.renewAPNsDeviceToken()` (and its analog on Android) under the hood of `Action.SendPushNotificationsToken` and remove FCM token fetching in common code?